.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+**Pull request recommendations:**
+- [ ] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
+- [ ] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
+- [ ] Provide context of changes.
```

Comment:
This will help me remember to include screenshots -- I always forget

```suggestion
- [ ] Provide description and context of changes, including any helpful screenshots.
```